person.rb

```diff
@@ -0,0 +1,38 @@
+require './nameable'
+require './decorators'
```

Comment:
- [ ] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages. Kindly ensure to improve upon this aspect.